### PR TITLE
BaSP-TG-41: List timesheets page

### PR DESCRIPTION
**Pull request by AlejandroSantini**

### CODE REVIEW

File: src/Components/TimeSheets/List/TimeSheetsList.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import TimeSheetItem from '../ListItem/ItemTimeSheets.jsx';
+import './list.module.css';
+
+const List = ({ list, deleteTimeSheet }) => {
+  console.log('array list', list);
```
Comment (GuidoCerioni) [inline src/Components/TimeSheets/List/TimeSheetsList.jsx:6]: Delete console.log

File: src/Components/TimeSheets/ListItem/ItemTimeSheets.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './item-time-sheets.module.css';
+
+const TimeSheetItem = ({ listItem, deleteTimeSheet }) => {
+  console.log('Time sheets list', listItem);
```
Comment (GuidoCerioni) [inline src/Components/TimeSheets/ListItem/ItemTimeSheets.jsx:5]: Delete console.log

**Review by GuidoCerioni — CHANGES_REQUESTED** [review]
You should delete console.logs

**Review by HiginiaMed — APPROVED** [review]
Good job!

File: src/Components/TimeSheets/index.js [hunk 3]
```diff
@@ -1,11 +1,47 @@
...
+      const responseJson = await response.json();
+      setList(responseJson.data);
+    } catch (error) {
+      console.error(error);
+    } finally {
+      setLoading(false);
+    }
+  }, []);
+
+  // API request to delete admins
```
[the remark this answers is not in the document]
Comment (AlejandroSantini) [reply src/Components/TimeSheets/index.js]: Comments are from an old commit

**Review by GuidoCerioni — APPROVED** [review]
Nice job!

**Review by Gearyandres94 — APPROVED** [review]
Looks Great!!

**Review by Facund0Fernandez — COMMENTED** [review]
The edit button redirects home. This is correct?

**Review by Facund0Fernandez — APPROVED** [review]

**Comment by AlejandroSantini** [comment]
> The edit button redirects home. This is correct?

The edit button redirects to <a href={`/time-theets/form?id=${listItem._id}`}>{pencil}</a>, I hope I understood your comment well.

**Pull request merged**